include/pacmod_game_control/publish_control.h

```diff
@@ -35,36 +36,31 @@ class PublishControl
   JoyAxis steering_axis = JoyAxis::LEFT_STICK_LR;
   float max_rot_rad = MAX_ROT_RAD_DEFAULT;
```

Comment:
some variables are `float` and some of them are `double`. Can we keep them the same?

Reply:
Anything joystick-related comes in as floats from the joy ROS driver. For some reason pacmod_msgs commands and reports are doubles even though the CAN interface is 16-bit scaled integers. I'll change it all to float and consider updating the ROS message types in the future from `float64` to `float32`.